client/src/model/states/model/geo_data.ts

```diff
@@ -1,9 +1,24 @@
+/**
+ * Represent the granularity of a geo query.
+ *
+ * Must match mapbox.
+ * TODO: Unify with geographic level somehow.
+ */
+enum GeoType {
+  address = 'address',
+  postcode = 'postcode',
```

Comment:
is this zip? if so can we call it zipcode

Reply:
This is annoying but it has to match the mapbox place types. I can't find the documentation for it but the result that geocoder returns has a field called "place_type" and when it's zipcode it's called a postcode and when it's address it's address. I need to check what county / city / state produces